models/stg_sage_intacct__ap_bill.sql

```diff
@@ -52,3 +53,4 @@ final as (
 )
 
 select * from final
+where _fivetran_deleted = false
```

Comment:
What if the `_fivetran_deleted` column is `null`? Then we will be filtering out all the records.

We want to provide a safety net in case this field is not populated. Below is how we should approach this:
```suggestion
where not coalesce(_fivetran_deleted, false)
```

Reply:
This allows for us to set the field to `false` if it is not populated, thus not removing the field erroneously.